src/viewProviders/resources.test.ts

```diff
@@ -31,7 +32,7 @@ describe("ResourceViewProvider methods", () => {
 
   it("getTreeItem() should return a CCloudEnvironmentTreeItem for a CCloudEnvironment instance", () => {
```

Comment:
This test seems wonky now.

Reply:
Agreed; updated in https://github.com/confluentinc/vscode/pull/654/commits/583cdfcbe5a46ba148d13d56c05855f0d93b7bf7